test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperOutputTest.cs

```diff
@@ -79,13 +79,14 @@ public void GenerateStartTag_ReturnsFullStartTag()
                 {
                     { "class", "btn" },
                     { "something", "   spaced    " }
-                });
+                },
+                htmlEncoder: new TestHtmlEncoder());
```

Comment:
We don't want to use the default encoder for unit testing. https://github.com/aspnet/Mvc/pull/2076#discussion-diff-25388824